webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -78,21 +81,39 @@ const HostDetails = ({
           <br />
           <br />
           <Grid>
-            <GridItem span={3}>
-              <Title headingLevel="h5" size="2xl">
-                {/* TODO: Make a generic Skeleton HOC (withSkeleton) */}
-                {response.name || <Skeleton />}{' '}
-                <HostGlobalStatus hostName={id} />
-              </Title>
+            <GridItem span={9}>
+              <SkeletonLoader status={status || STATUS.PENDING}>
+                {response && (
+                  <>
+                    <div className="hostname-wrapper">
+                      <SkeletonLoader status={status || STATUS.PENDING}>
+                        {response && (
+                          <Title
+                            className="hostname-truncate"
+                            headingLevel="h5"
+                            size="2xl"
+                          >
+                            {response.name}
+                          </Title>
+                        )}
+                      </SkeletonLoader>
+                    </div>
+                    <Split style={{ display: 'inline-flex' }} hasGutter>
+                      <SplitItem>
+                        <HostGlobalStatus hostName={id} />
+                      </SplitItem>
+                      <SplitItem>
+                        <Badge> {response?.operatingsystem_name}</Badge>
```

Comment:
Probably uneccessary, but doesn't hurt, so I'll merge unless I find something else :)
```suggestion
                        <Badge> {response.operatingsystem_name}</Badge>
```